src/enums/views.ts

```diff
@@ -6,12 +6,12 @@ export enum Views {
   SIGN_IN = 'signin',
   SIGN_OUT = 'signout',
 
-  // Amalgamation views
-  AMALGAMATION_REG_BUSINESS_INFO = 'amalg-reg-business-info',
-  AMALGAMATION_REG_INFORMATION = 'amalg-reg-information',
-  AMALGAMATION_REG_PEOPLE_ROLES = 'amalg-reg-people-roles',
-  AMALGAMATION_REG_REVIEW_CONFIRM = 'amalg-reg-review-confirm',
-  AMALGAMATION_REG_SHARE_STRUCTURE = 'amalg-reg-share-structure',
+  // Amalgamation (regular and short-form) views
+  AMALGAMATION_BUSINESS_INFO = 'amalgamation-business-info',
+  AMALGAMATION_INFORMATION = 'amalgamation-information',
+  AMALGAMATION_PEOPLE_ROLES = 'amalgamation-people-roles',
+  AMALGAMATION_REVIEW_CONFIRM = 'amalgamation-review-confirm',
+  AMALGAMATION_SHARE_STRUCTURE = 'amalgamation-share-structure', // regular only
```

Comment:
Same views for both types of amalgamations, but different routes/steps/resources.